api/nodes/views.py

```diff
@@ -1997,7 +1997,7 @@ def get_object(self):
 
 
 class NodeInstitutionRelationship(JSONAPIBaseView, generics.RetrieveUpdateAPIView, NodeMixin):
```

Comment:
This view should be read only for registrations--is this true? I don't see a test verifying this.